src/raftstore/store/worker/apply.rs

```diff
@@ -401,8 +406,11 @@ impl<'a> ApplyContextCore<'a> {
     }
 }
 
+/// ApplyContext is the context for a region apply work.
```

Comment:
Looks like in Rust world we just `/// The context ... ` instead of `/// Xxx is the context... `.

Reply:
Maybe better to explain where this context will be used